edb/pgsql/metaschema.py

```diff
@@ -3444,6 +3444,53 @@ def __init__(self) -> None:
         )
 
 
+class PostgresJsonConfigValueToGelConfigValueFunction(
+    trampoline.VersionedFunction,
+):
+    """Convert a Postgres config value to Gel config value.
```

Comment:
I think my inclination is to keep calling it "edgedb" as a "codename" in internal code like this, but I don't feel strongly.

Just makes it a little easier to be consistent in function names and stuff, though obviously it is kind of fucked no matter what.

Reply:
We also have the `backend` (postgres) and `frontend` (edgedb-server) used around (which I think I prefer)